File: src/java/com/ijse/wearit/dto/ItemDetailsDTO.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.ijse.wearit.dto;

import org.springframework.stereotype.Component;

/**
 *
 * @author ABC
 */
@Component
public class ItemDetailsDTO extends SuperDTO{

    private Integer _id;
    private String _sizeName;
    private String _itemDescription;
    private double _unitPrice;
    private int _qtyOnHand;

    public Integer getId() {
        return _id;
    }

    public void setId(Integer id) {
        this._id = id;
    }

    public String getSizeName() {
        return _sizeName;
    }

    public void setSizeName(String sizeName) {
        this._sizeName = sizeName;
    }

    public String getItemDescription() {
        return _itemDescription;
    }

    public void setItemDescription(String itemDescription) {
        this._itemDescription = itemDescription;
    }

    public double getUnitPrice() {
        return _unitPrice;
    }

    public void setUnitPrice(double unitPrice) {
        this._unitPrice = unitPrice;
    }

    public int getQtyOnHand() {
        return _qtyOnHand;
    }

    public void setQtyOnHand(int qtyOnHand) {
        this._qtyOnHand = qtyOnHand;
    }  
}

File: web/resources/js/js-a/ItemDetailsDTO.js
/* 
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
function ItemDetailsDTO() {
    var _id;
    var _sizeName;
    var _itemDescription;
    var _unitPrice;
    var _qtyOnHand;

    
     this.getId=function () {
         return this._id;
     },
    this.getSizeName=function () {
        return this._sizeName;
    },
    this.getItemDescription=function () {
        return this._itemDescription;
    },
    this.getUnitPrice=function () {
        return this._unitPrice;
    },
    this.getQtyOnHand=function () {
        return this._qtyOnHand;
    },
    
    this.setId=function (id) {
         this._id=id;
    },
    this.setSizeName=function (sizeName) {
         this._sizeName=sizeName;
    },
    
    this.setItemDescription=function (itemDescription) {
         this._itemDescription=itemDescription;
    },
    this.setUnitPrice=function (unitPrice) {
         this._unitPrice=unitPrice;
    },
    this.setQtyOnHand=function (qty) {
         this._qtyOnHand=qty;
    };

}

